tests/test_reference_samplers.py

```diff
@@ -94,16 +94,20 @@ def test_track_data(self):
 
         sampleset = LatticeLNLSSampler().sample(
             bqm=bqm, problem_dims=(2,2,2), num_reads=num_reads,
-            qpu_sampler=MockDWaveSamplerGeneralization(), topology='cubic',max_iter=max_iter,
+            qpu_sampler=MockDWaveSampler(), topology='cubic',max_iter=max_iter,
             qpu_params=dict(chain_strength=2), reject_small_problems=False,
-            track_data=False
+            track_qpu_branch=False
             )
 
+        self.assertEquals(False, 'tracked_samples' in sampleset.info)
+        self.assertEquals(False, 'tracked_subsamples' in sampleset.info)
+        self.assertEquals(False, 'tracked_subproblems' in sampleset.info)
```

Comment:
You might want to use [`assertFalse`](https://apps.docusign.com/send/home) or even better [`assertNotIn`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertNotIn) for this. One good reason to use a more specific assertion, generally, is better/more specific error message.